client/allocrunner/taskrunner/logmon_hook.go

```diff
@@ -58,8 +60,8 @@ func (*logmonHook) Name() string {
 	return "logmon"
 }
 
-func (h *logmonHook) launchLogMon() error {
-	l, c, err := logmon.LaunchLogMon(h.logger)
```

Comment:
Did you mean to pass nil in here?

Reply:
Oops nope, missed this when I was refactoring for the PR. Fixed! (I had separate paths for launch/reattach when I was testing)